operator/Makefile

```diff
@@ -46,9 +46,13 @@ licenses: ## Verifies dependency licenses and requires GITHUB_TOKEN to be set
 
 apply: ## Apply the controller into your ~/.kube/config cluster
 	KO_DOCKER_REPO=$(CONTAINER_IMAGE_REGISTRY)/kit ko apply --bare -f config/
```

Comment:
Do you need this line, too?

Reply:
I wanted to separate the control plane CRD yaml spec from the controller. So config/ contains this spec, controller and webhook are subdirectories that contain their specific yaml files.